src/Tizen.Applications.Common/Tizen.Applications/RecentApplicationInfo.cs

```diff
@@ -65,15 +65,53 @@ public class RecentApplicationInfo : ApplicationInfo
         /// <since_tizen> 3 </since_tizen>
         public RecentApplicationControl Controller { get; private set; }
 
-        internal RecentApplicationInfo(Interop.ApplicationManager.RuaRec record) : base(Marshal.PtrToStringAnsi(record.pkgName))
+        /// <summary>
+        /// Gets the recent application package name.
+        /// </summary>
+        /// <since_tizen> 10 </since_tizen>
+        public string PackageName { get; private set; }
```

Comment:
Please check API Version. I think it should be '13'.

Reply:
I've update it.
Thanks